cli_tools/gce_vm_image_export/exporter/exporter_test.go

```diff
@@ -113,11 +115,33 @@ func TestBuildDaisyVarsWithSimpleImageName(t *testing.T) {
 		ws+format+ws,
 		ws+network+ws,
 		ws+subnet+ws,
-		ws+"aRegion"+ws)
+		ws+"aRegion"+ws,
+		"")
 
 	assert.Equal(t, "global/images/anImage", got["source_image"])
 }
 
+func TestBuildDaisyVarsWithComputeServiceAccount(t *testing.T) {
+	resetArgs()
+	ws := "\t \r\n\f\u0085\u00a0\u2000\u3000"
+	got := buildDaisyVars(
+		"", "", "", "", "", "",
+		ws+"account1"+ws)
+
+	assert.Equal(t, "account1", got["compute_service_account"])
+}
+
+func TestBuildDaisyVarsWithoutComputeServiceAccount(t *testing.T) {
+	resetArgs()
+	ws := "\t \r\n\f\u0085\u00a0\u2000\u3000"
+	got := buildDaisyVars(
+		"", "", "", "", "", "",
+		ws)
+
+	_, o= got["compute_service_account"]
+	assert.False(t, ok)
```

Comment:
o or ok? Does this even compile?

Consider a better name, e.g. hasComputeServiceAccountVar instead of o/ok

Reply:
used hasVar